PowerShellGet/PSModule.psm1

```diff
@@ -5057,7 +5067,7 @@ function New-ScriptFileInfo
 
         if(-not $Version)
         {
-            $Version = [Version]'1.0'
+            $Version = '1.0'
         }

```

Comment:
This is for if($version) case.
Earlier, PS runtime validates the version value as parameter type was [Version]. Since this param type is string now, we should validate this param value early in this function.